stacks/spica/src/function/interface.ts

```diff
@@ -137,6 +137,25 @@ export function emptyWebhook(): Webhook {
   };
 }
 
+export interface WebhookLog {
+  _id: string;
+  webhook: string;
+  request: object;
+  response: object;
+  execution_time: Date;
```

Comment:
Could you generate this from `_id` by using aggregation?

Reply:
It's client side.